exercises/practice/secret-handshake/secret-handshake.rakutest

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.dirname;
+use SecretHandshake;
+
+cmp-ok( # begin: b8496fbd-6778-468c-8054-648d03c4bb23
+    handshake(1),
+    "eq",
```

Comment:
`eq` for matching a list? Do you mean to use `~~`?

Reply:
Looks like `prove6` is missing from the Windows docker file (again?) :

~~~text
The term 'prove6' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
~~~